docs/release-notes/release-notes-0.4.0.md

```diff
@@ -0,0 +1,509 @@
+# DCRDEX v0.4.0
+
+Dec 23, 2021
```

Comment:
date of the `v0.4.0-rc1` tag.  I suppose we'll update this date as we get to final.